src/shared/string-avatar.ts

```diff
@@ -1,27 +1,28 @@
-function stringToColor(string: string) {
-	let hash = 0;
-	let i;
+import { UserService } from 'clients/CoreService';
+import { useMemo } from 'react';
+import { useQuery } from 'react-query';
 
-	/* eslint-disable no-bitwise */
-	for (i = 0; i < string.length; i += 1) {
-		hash = string.charCodeAt(i) + ((hash << 5) - hash);
-	}
+/**
+ * Upload file to S3 with previously received pre-signed POST data.
+ * @param postUrl
+ * @param file
+ * @returns {Promise<void>}
+ */
+export const GetUserIconUrl = (id: number): string => {
+	const { data: content, isLoading: isContentLoading } = useQuery('users/icon', () =>
```

Comment:
Create entity userIcon in `consts/entities`:
```suggestion
	const { data: content, isLoading: isContentLoading } = useQuery(entities.userIcon, () =>
```

Reply:
entities.avatar